Common/TableProducer/PID/pidTPC.cxx

```diff
@@ -68,18 +68,20 @@ struct tpcPid {
   o2::pid::tpc::Response* responseptr = nullptr;
   // Network correction for TPC PID response
   Network network;
+  std::string networkPathOnAlien = ""; // Will be set by the user or automatically if TPCPIDResponse object is fetched from CCDB
 
   // Input parameters
   Service<o2::ccdb::BasicCCDBManager> ccdb;
   Configurable<std::string> paramfile{"param-file", "", "Path to the parametrization object, if emtpy the parametrization is not taken from file"};
   Configurable<std::string> url{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
   Configurable<std::string> ccdbPath{"ccdbPath", "Analysis/PID/TPC/Response", "Path of the TPC parametrization on the CCDB"};
   Configurable<long> ccdbTimestamp{"ccdb-timestamp", 0, "timestamp of the object used to query in CCDB the detector response. Exceptions: -1 gets the latest object, 0 gets the run dependent timestamp"};
+  o2::ccdb::CcdbApi ccdb_api; /// API to access CCDB headers
```

Comment:
So you need the api to access the headers? And you cannot use the `BasicCCDBManager` `ccdb` right?

Reply:
e.g. as in https://github.com/AliceO2Group/O2Physics/blob/master/Common/TableProducer/timestamp.cxx#L35